cmd/controlplane/external_webhooks.go

```diff
@@ -78,7 +78,7 @@ func (o *externalWebhooksServerOptions) run(ctx context.Context) error {
 	}
 
 	if err = kargoapi.AddToScheme(cluster.GetClient().Scheme()); err != nil {
-		return fmt.Errorf("failed to add custom kargo type to runtime scheme: %w", err)
+		return fmt.Errorf("error adding Kargo API to Kargo manager scheme: %w", err)
```

Comment:
The controller uses multiple managers and clients, so its error messages differentiate between errors setting up the scheme for one vs the other. We don't need that differentiation here.

```suggestion
		return fmt.Errorf("error adding Kargo API to scheme: %w", err)
```